sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_models.py

```diff
@@ -212,4 +212,14 @@ class RestoreOperation(_Operation):
     """
 
 
-SelectiveKeyRestoreOperation = RestoreOperation
+class SelectiveKeyRestoreOperation(_Operation):
+    """A Key Vault operation restoring a single key.
+
+    :ivar str status: status of the backup operation
+    :ivar str status_details: more details of the operation's status
+    :ivar error: Error encountered, if any, during the operation
+    :type error: ~key_vault_client.models.Error
```

Comment:
don't think this typing would render properly

Reply:
Right you are. I copied it from the generated model. Fixing it depends on how we expose the error. I made this part of #13575.